.github/crd-config/config.yaml

```
@@ -1,3 +1,22 @@
+processor:
+  # RE2 regular expressions describing types that should be excluded from the generated documentation.
+  ignoreTypes:
+    - "User$"
+    - "UserList$"
+    - "UserSpec$"
+    - "UserStatus$"
+    - "UserAuthenticationSpec$"
+    - "UserAuthorizationSpec$"
+    - "UserTemplateSpec$"
+    - "Password$"
+    - "PasswordSource$"
+    - "ACLRule$"
+    - "ACLType$"
+    - "ACLOperation$"
+    - "PatternType$"
+    - "ResourceType$"
+    - "ACLResourceSpec$"
+    - "ClusterSource$"
```

```suggestion
    - "ClusterSource$"
    - "AdminAPISpec$"
    - "AdminSASL$"
    - "MetadataTemplate$"
```
A few more only used in User stuff currently.

---

thank you!